pkg/ctrl/server/fd.go

```diff
@@ -13,22 +13,24 @@
 // limitations under the License.
 //
 
-package graph
+package server
 
 import (
 	"context"
 	"fmt"
 	"strings"
 
-	"github.com/chaos-mesh/chaos-mesh/pkg/ctrlserver/graph/model"
+	"github.com/chaos-mesh/chaos-mesh/pkg/bpm"
+	"github.com/chaos-mesh/chaos-mesh/pkg/ctrl/server/model"
 )
 
 // GetFdsOfProcess returns fd-target pairs
-func (r *Resolver) GetFdsOfProcess(ctx context.Context, process *model.Process) ([]*model.Fd, error) {
+func (r *Resolver) GetFdsOfProcess(ctx context.Context, process *model.Process) []*model.Fd {
```

Comment:
why not return error?

Reply:
Because the errors often occur on some short-life process.

Reply:
I will leave some comments to explain it.

Reply:
Left.